pytorch_toolkit/ote/ote/readme.md

```diff
@@ -0,0 +1,201 @@
+# Model Templates
+
+Model Templates defines training procedure and its interface for a given neural network topology.
+
+## Directory structure (openvino_training_extensions/pytorch_toolkit)
+
+Each model template is related to 4 scripts: `train.py`, `eval.py`, `export.py` and `quantize.py`.
+
+```bash
+├── object_detection
+│   ├── face-detection
+│   │   ├── face-detection-0200
+│   │   │   ├── config.py
+│   │   │   └── template.yml
+│   │   ├── face-detection-0202
+│   │   │   ├── config.py
+│   │   │   └── template.yml
+│   │   ├── problem.yml
+│   │   ├── schema.json
+│   │   └── tools
+│   │       ├── eval.py
+│   │       └── train.py
+│   ├── horizontal-text-detection
+│   │   ├── horizontal-text-detection-0001
+│   │   │   ├── config.py
+│   │   │   └── template.yml
+│   │   ├── problem.yml
+│   │   ├── schema.json
```

Comment:
What is `problem.yml` and `schema.json`?

Reply:
Now there are inside common file https://github.com/openvinotoolkit/training_extensions/blob/model_template/pytorch_toolkit/problems.yaml